src/asm/parser.y

```diff
@@ -1214,8 +1220,18 @@ reloc_16bit_no_str : relocexpr_no_str {
 				warning(WARNING_TRUNCATION, "Expression must be 16-bit\n");
 			$$ = $1;
 		}
+		| inline_fragment { rpn_Symbol(&$$, $1); }
 ;
 
+inline_fragment	: T_2LBRACK {
+			out_PushInlineFragmentSection();
+			sprintf($<inlineFragmentName>$, "$%" PRIu32, inlineFragmentID++);
```

Comment:
Either this or the `%type`, but not both, please. I personally prefer the `%type`.

Reply:
I'm changing it to `T_2LBRACK <inlineFragmentName>{ ... } asmfile T_2RBRACK { ... }`, using [this Bison feature](https://www.gnu.org/software/bison/manual/html_node/Typed-Mid_002dRule-Actions.html#Typed-Mid_002dRule-Actions), but it does need that one `<inlineFragmentName>` for the mid-action rule.